lib/capistrano/tasks/wp.cap

```diff
@@ -12,6 +24,46 @@ namespace :wp do
 
   namespace :setup do
 
+    # Generates a random password and stores it as :wp_pass
+    task :generate_password do
+      o = [('a'..'z'), ('A'..'Z')].map { |i| i.to_a }.flatten
+      password = (0...18).map { o[rand(o.length)] }.join
+      set :wp_pass, password
+    end
+
+    # Installs WordPress
+    task :install
+      config = Config.new
+      execute :wp, "core install --url='#{config.url}' --title='#{config.title}' --admin_user='#{config.user}' --admin_password='#{config.password}' --admin_email='#{config.email}' --path='wp'"
+    end
+
+    # Prints report for the user
+    task :report do
+      config = Config.new
+
+      puts <<-MSG
+      \e[32m
+      =========================================================================
+        WordPress has successfully been installed. Here are your login details:
+
+        Username:       #{config.user}
+        Password:       #{config.password}
+        Email address:  #{config.email}
+        Log in at:      #{config.url}/wp/wp-admin
+      =========================================================================
+      \e[0m
+      MSG
+    end
+
+    # Creates wp-config.php locally
+    task :generate_local_files do
```

Comment:
Those are both used in the wp-config.php.erb file in the templates folder. That is used to write WP Constants to the config file.